client/my-sites/plugins/minimum-jet-pack-version-notice.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import notices from 'notices';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	getSite,
+	isJetpackSite,
+	siteHasMinimumJetpackVersion
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends React.Component {
+
+	static propTypes = {
+		translate: React.PropTypes.func.isRequired,
+		minimumJetpackVersionFailed: React.PropTypes.bool.isRequired,
+		site: React.PropTypes.object,
+	};
+
+	notificationHandler( { minimumJetpackVersionFailed, site, translate } ) {
```

Comment:
This method is always used with the current props, so we don't need to pass them as parameters - we can import them inside instead. Also, can we be a little more specific about the method name - would something like `showWarning` or similar could be more reasonable?